dubbo-common/src/test/java/org/apache/dubbo/common/utils/AnnotationUtilsTest.java

```diff
@@ -173,31 +174,38 @@ public void testGetMetaAnnotations() {
         assertEquals(Inherited.class, metaAnnotations.get(0).annotationType());
 
         metaAnnotations = getMetaAnnotations(Service.class);
+        List<String> testAnnotations = new ArrayList<>();
+        for(int i = 0; i < metaAnnotations.size(); i++) {
```

Comment:
You can consider using the enhanced for loop or even java.util.stream to improve code readability.